homeassistant/components/incomfort/water_heater.py

```diff
@@ -81,7 +84,7 @@ def supported_features(self):
     def current_operation(self):
         """Return the current operation mode."""
         if self._heater.is_failed:
-            return "Failed ({})".format(self._heater.fault_code)
+            return "Fault code: {}".format(self._heater.fault_code)
```

Comment:
🤷‍♂ 

I guess you're not making it _worse_. But really current operation should be standardized.

Reply:
The state property is only to be defined by the abstract base class..

Reply:
Yes, I guess I was being a bit cheeky. -Thanks for merging.